.github/workflows/ci.yml

```diff
@@ -143,12 +145,6 @@ jobs:
           key: ${{ runner.os }}-node-${{ matrix.NODE_VERSION }}-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
               ${{ runner.os }}-node-${{ matrix.NODE_VERSION }}-
-      - name: Install dependencies (Node < 10)
```

Comment:
Unrelated change, this should be done when removed node 12

Reply:
I am gonna put it back